src/Umbraco.Web/Trees/MemberGroupTreeController.cs

```diff
@@ -19,5 +19,13 @@ protected override IEnumerable<TreeNode> GetTreeNodesFromService(string id, Form
                 .OrderBy(x => x.Name)
                 .Select(dt => CreateTreeNode(dt.Id.ToString(), id, queryStrings, dt.Name, "icon-item-arrangement", false));
         }
+
+        protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
```

Comment:
same goes for this 😄

Reply:
-> https://github.com/umbraco/Umbraco-CMS/pull/3398